spring-cloud-aws-samples/spring-cloud-aws-sns-sample/infrastracture/README.md

```diff
@@ -0,0 +1,61 @@
+# Spring Cloud AWS SNS Sample App Infrastructure
+
+This sample shows sending SNS messages and receiving them with SQSListener and NotificationMapping. 
+To use NotificationMapping you will need to use NGROK: https://ngrok.com/docs
+Firstly you will need to start 'ngrok http 127.0.0.1:{YourPort}' after that you will need to change URL inside CDK.
+Once you changed URL inside CDK you can run CDK deploy. Subscription will be automatically sent to NGROK url.
+There is possibility subscription isn't confirmed since application is not started as of CDK deploy, so you should resend subscription from a console.
```

Comment:
"console" can be understood as a terminal or AWS console. Maybe we can be more precise here and give step by step instructions how to do it?